pages/validators/running_validator.md

```diff
@@ -1,3 +1,3 @@
 # Running a Validator
 
-TODO: this page is under construction. In the meantime, please refer to genesis validator [instructions](../networks/network1/genesis/starting_the_network.md).
\ No newline at end of file
+TODO: this page is under construction.
```

Comment:
Does this page actually need anything? Kinda feel like the information just already lives in other pages. We can probably delete it for now and re-create it if addtl information is needed